### PR TITLE
Error with missing scopes + add check over cabundle file + proxy cert flag

### LINKED ISSUE #34: error 500 uploading user certificate when no CA is there
this should make sure to produce a sentry message and also return something useful to the user

```python
>>> import ctadata
>>> cl = ctadata.api.APIClient()
>>> cl.upload_certificate("/tmp/x509up_u1000")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/savchenk/ctadata/ctadata/api.py", line 158, in upload_certificate
    raise CertificateError(r.text)
ctadata.api.CertificateError: <!doctype html>
<html lang=en>
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>


```

```python
  File "/downloadservice/downloadservice/app.py", line 162, in decorated
    return f(user, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/downloadservice/downloadservice/app.py", line 190, in upload_cert
    cabundle = open(app.config['CTADS_CABUNDLE'], 'r').read()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/downloadservice-data/dcache_cabundle.pem'

```

### PR DESCRIPTION
**Pull request by Ph0tonic**
Fixes #34

### CODE REVIEW

File: downloadservice/app.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def decorated(*args, **kwargs):
                 user = auth.user_for_token(token)
                 if user is not None and not auth.check_scopes(
                         'access:services!service=downloadservice', user):
-                    user = None
+                    return 'Access denied, token scopes are insuficient', 403
```
Comment (volodymyrss) [inline downloadservice/app.py:157]:
```suggestion
                    return 'Access denied, token scopes are insufficient. If you need access to this service, please contact CTA-CH DC team at EPFL.', 403
```

**Comment by Ph0tonic** [comment]
I deployed this new version 👍

**Comment by volodymyrss** [comment]
I still get 500 in invalid CA. Why there is no sentry entry?..

**Comment by Ph0tonic** [comment]
I haven't resolved this one for now:
- https://github.com/cta-epfl/downloadservice/issues/32

I will focus on it now.

**Pull request merged**